metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -45,6 +50,7 @@
 SNAPSHOT_TABLE_REGEX = re.compile(r"^(.+)@(\d{13})$")
 
 BQ_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+BQ_DATE_SHARD_FORMAT = "%Y%m%d"
```

Comment:
this is to support date sharded (or suffixed, so to speak) bigquery audit log tables